src/components/datasets/DatasetForm.js

```diff
@@ -18,38 +20,53 @@ const validateJson = (rule, value) => {
     }
 };
 
-const DatasetForm = ({ initialValue, formRef }) => {
+const DatasetForm = ({ initialValue, form, updateMode}) => {
+    const discoveryValidator = useDiscoveryValidator();
     return (
-        <Form ref={formRef} layout="vertical">
+        <Form form={form} layout="vertical" initialValues={initialValue}>
             <Item
                 label="Title"
                 name="title"
-                initialValue={initialValue?.title || ""}
+                // initialValue={initialValue?.title || ""}
                 rules={[{ required: true }, { min: 3 }]}
             >
                 <Input placeholder="My Dataset" size="large" />
             </Item>
             <Item
                 label="Description"
                 name="description"
-                initialValue={initialValue?.description || ""}
+                // initialValue={initialValue?.description || ""}
```

Comment:
same q here

Reply:
Removed the commented code. Initial values are handled by the form object using the Item's name, except for the two DropBoxJsonSelect components, which has its own initial value mechanism.